### DISCUSSION TITLE
OCLP booting issues

### DISCUSSION THREAD

**Discussion by geogyinjesus** [community/community#124829] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi
I spend more than 24 hours to install OCPL Sonoma on my macpro 5,1 2012 RX580.
The main issue is I can’t see the boot picker.
After installing and pressing the option keys I can hear chimes the I can hear the graphics card fans on then off, again on after a minute off; it continues non stop.
I can’t see anything on my screen.
Kindly help me to install Sonoma.
Perhaps my graphics card is not flashed and there’s no game mode switch. Also I can’t do sip.😩🥲

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @geogyinjesus !  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
